app/views/govuk_publishing_components/component_guide/index.html.erb

```diff
@@ -2,12 +2,10 @@
 
 <div class="component-markdown">
   <p>Components are packages of template, style, behaviour and documentation that live in your application.</p>
-  <p>A component must:</p>
   <ul>
-    <li><a href="https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_principles.md">meet the component principles</a></li>
-    <li><a href="https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md">follow component conventions</a></li>
+    <li><a href="https://docs.publishing.service.gov.uk/manual/components.html">all documentation for components</a></li>
+    <li>how to <a href="https://github.com/alphagov/govuk_publishing_components/blob/master/docs/publishing-to-rubygems.md">release a new version of the gem</a></li>
```

Comment:
This is tightly coupled with the repo (you create a pull request for your change via GitHub, you create a release via GitHub, I don't see how the components guide fits in this workflow) and we link to it from the main readme file.

Reply:
That's all true, but as I said there's a lot of people who don't know this procedure and keep getting it wrong. Developers know where the component guide is, so I'm just trying to provide this as even clearer signposting (yes, it's pretty easy to find from the repo, but I think our procedure is perhaps different to what developers are used to, which is why they don't look for it).

Reply:
As discussed, let's get this in and keep an eye on the next releases (let's say by the end of the year) if they improve, we'll keep it, otherwise we'll remove it and look at other ways to communicate this at the community level.

Reply:
Thanks!